src/System.CommandLine/Help/HelpBuilder.Default.cs

```diff
@@ -50,63 +50,46 @@ public static string GetArgumentDefaultValue(Argument argument)
             {
-                if (argument.FirstParent?.Symbol is Command)
-                {
-                    return $"<{argument.Name}>";
-                }
-                else
-                {
-                    return "";
-                }
+                return $"<{argument.HelpName}>";
             }
+            else if (!argument.IsBoolean() && argument.CompletionSources.Count > 0)
+            {
+                IEnumerable<string> completions = argument
+                    .GetCompletions(CompletionContext.Empty)
+                    .Select(item => item.Label);
 
-            string firstColumn;
-            var completions = argument
-                .GetCompletions(CompletionContext.Empty)
-                .Select(item => item.Label)
-                .ToArray();
-
-            var arg = argument;
-            var helpName = arg?.HelpName ?? string.Empty;
+                string joined = string.Join("|", completions);
 
-            if (!string.IsNullOrEmpty(helpName))
-            {
-                firstColumn = helpName!;
-            }
-            else if (completions.Length > 0)
-            {
-                firstColumn = string.Join("|", completions);
-            }
-            else
-            {
-                firstColumn = argument.Name;
+                if (!string.IsNullOrEmpty(joined))
+                {
+                    return $"<{joined}>";
+                }
             }
 
-            if (!string.IsNullOrWhiteSpace(firstColumn))
-            {
-                return $"<{firstColumn}>";
-            }
-            return firstColumn;
+            // By default Option.Name == Argument.Name, don't repeat it
+            return argument.FirstParent?.Symbol is not Option ? $"<{argument.Name}>" : "";
         }
 
-        /// <summary>
-        /// Gets the description for the specified symbol (typically the used as the second column in help text).
-        /// </summary>
-        /// <param name="symbol">The symbol to get the description for.</param>
-        public static string GetIdentifierSymbolDescription(IdentifierSymbol symbol) => symbol.Description ?? string.Empty;
-
         /// <summary>
         /// Gets the usage label for the specified symbol (typically used as the first column text in help output).
         /// </summary>
         /// <param name="symbol">The symbol to get a help item for.</param>
-        /// <param name="context">The help context, used for localization purposes.</param>
         /// <returns>Text to display.</returns>
-        public static string GetIdentifierSymbolUsageLabel(IdentifierSymbol symbol, HelpContext context)
+        public static string GetIdentifierSymbolUsageLabel(Command symbol)
+            => GetIdentifierSymbolUsageLabel(symbol, symbol._aliases);
+
+        /// <inheritdoc cref="GetIdentifierSymbolUsageLabel(Command)"/>
+        public static string GetIdentifierSymbolUsageLabel(Option symbol)
```

Comment:
Should the word "identifier" be removed from these two methods? Since the type was removed, the term might no longer have significance in the API.

Reply:
TBH I expect that we are going to remove it very soon and make help more customizable in V2.

Reply:
I've renamed it.